tests/behave/Dockerfile

```diff
@@ -12,6 +12,6 @@ USER seluser
 COPY requirements.txt ./
 RUN pip3 install --user -r requirements.txt
 
-COPY . .
+COPY --chown=seluser . .
```

Comment:
Why do we need this change?

Reply:
The geckodriver.log file seems to get owned by root:root when building the Dockerfile alone.

Reply:
After speaking to @sldblog offline I have decided to remove `--chown=seluser ` addition as it's not supported in all  docker versions